src/core/components/array-model.jsx

```diff
@@ -43,15 +43,18 @@ export default class ArrayModel extends Component {
     */
 
     return <span className="model">
-      <ModelCollapse title={titleEl} expanded={ depth <= expandDepth } collapsedContent="[...]">
-        [
-          {
+      <ModelCollapse title={titleEl} expanded={depth <= expandDepth} collapsedContent="[...]">
+        {/* OutSystems change - display the Array Model in a different manner Array[object] Object: object content */}
+        Array[object]<p></p>
+        <p>object:</p>
+        {
             properties.size ? properties.entrySeq().map( ( [ key, v ] ) => <Property key={`${key}-${v}`} propKey={ key } propVal={ v } propClass={ propClass } />) : null
-          }
-          {
-            !description ? (properties.size ? <div className="markdown"></div> : null) :
-              <Markdown source={ description } />
-          }
+        }
+        { /* OutSystems change: remove description
```

Comment:
Any reason why we are hiding information?

Reply:
yes. 
![image](https://user-images.githubusercontent.com/72461192/123406793-31fd4700-d5a3-11eb-9048-1be362432877.png)

It was duplicating the description. We don't need the description in the Model

Reply:
from swagger editor:
![image](https://user-images.githubusercontent.com/72461192/123407872-586fb200-d5a4-11eb-97f7-266cf2945ca0.png)

Reply:
In that specific case, where the array is appearing directly in the input/output. But I was thinking in the case it's used inside another structure - then it might not appear at all with this change.

Reply:
You were right. I created a structure with an array inside and the description was not being included. Fixed